tests/test-definition.yaml

```
@@ -13,23 +13,27 @@ dimensions:
       - 1.27.0
   - name: nifi_new
     values:
-      - 2.2.0
-      # Alternatively, if you want to use a custom image, append a comma and the full image name to the product version
-      # as in the example below.
-      # - 2.2.0,oci.stackable.tech/sandbox/nifi:2.2.0-stackable0.0.0-dev
+      - 2.2.0 # oci.stackable.tech/sandbox/nifi:2.2.0-stackable0.0.0-dev
   - name: nifi-latest
     values:
-      - 2.2.0
-      # Alternatively, if you want to use a custom image, append a comma and the full image name to the product version
-      # as in the example below.
-      # - 2.2.0,oci.stackable.tech/sandbox/nifi:2.2.0-stackable0.0.0-dev
+      - 2.2.0 # oci.stackable.tech/sandbox/nifi:2.2.0-stackable0.0.0-dev
+  - name: nifi-iceberg
+    values:
+      # - 2.2.0 # oci.stackable.tech/sandbox/nifi:2.2.0-stackable0.0.0-dev
+      - 2.0.0,oci.stackable.tech/sandbox/sbernauer/nifi:2.2.0-stackable0.0.0-dev-with-iceberg
```

Switch once https://github.com/stackabletech/docker-images/pull/1060 is merged
```suggestion
      - 2.2.0 # oci.stackable.tech/sandbox/nifi:2.2.0-stackable0.0.0-dev
```